Docs/user_guide/image_segmentation.md

```diff
@@ -67,5 +67,6 @@ For more extensions related to segmentations, open the ["Segmentations" category
 ## Tutorials
 
 To get started, check out these pages:
-- [Segmentation tutorials](https://www.slicer.org/wiki/Documentation/Nightly/Training#Segmentation): step by step slide and video tutorials
+- [Tutorials](https://github.com/EBATINCA/SlicerTutorials/tree/main): repository of actively maintained tutorials
```

Comment:
Thanks for working on this :pray: 

Before we more broadly disseminate, could you instead consider forking the repository we created a few years for this purpose: https://github.com/Slicer/tutorials.slicer.org

References:
* https://github.com/Slicer/Slicer/issues/6676

Reply:
Did we create tutorials.slicer.org to list all the tutorials (training portal) or to store the tutorial files (slides and data), or both? It would make sense to store some fundamental tutorials in the Slicer github organization, while more specialized tutorials could be still listed in the training portal but would be stored and maintained externally.

Reply:
Yes my intention was to start a common repository where the new tutorials could go. But it has only the one tutorial in it so far, so I didn't want to remove the link to the wiki, where there are still many tutorials. Should we move the still usable tutorials to this repo and get rid of the wiki link altogether?

Reply:
Tutorials are already relatively heavyweight (large source files, many images, test data set, etc.) and they will be bigger when internationalization and automatic screenshot updates via github actions will be implemented (the tutorial will be automatically run via python scripting in all supported languages when Slicer or the tutorial changes). Therefore, we plan to have one tutorial per repository.

There are already a couple of tutorials hosted like this:
- https://github.com/Slicer/SlicerProgrammingTutorial
- https://github.com/Slicer/SlicerDataLoadingAndVisualizationTutorial
- https://spujol.github.io/SlicerVisualizationTutorial/
- https://spujol.github.io/SlicerDICOMTutorial/
- https://spujol.github.io/SlicerProgrammingTutorial/

Therefore, we should have a dedicated repository for this segmentation tutorial, too. We would like to host essential tutorials in the Slicer organization for easy maintenance along with Slicer core updates. Since this segmentation tutorial is quite fundamental, it would make sense to move it under the Slicer organization, but it is up to you: if you prefer to maintain yourself and review&merge pull requests, then you can keep it under your organization; if you would prefer to hand off its maintenance and updates to the community and Slicer core developers, then you can transfer its ownership to the `Slicer` organization.

We are working on putting together a training page with @jcfr. It will have the same design as slicer.org. You can have a look at its current content [here](https://hackmd.io/yjTx5oIxRiyvuXUPSpoPCw) (still work in progress).

Reply:
The page looks great! I don't mind either keeping or transferring ownership. You say it is so basic that it should belong to the Slicer organization. In any case it seems there will be more things to do than just the ownership (renaming the file, standard folders/locations for the automatic translators etc.), so once we have the list of these I'll make the changes.